src/components/Dashboard/TodoList.vue

```diff
@@ -1,31 +1,26 @@
 <template>
   <div id="todo-list">
-    <add-comment-dialog
+    <AddCommentDialog
       :dialog="addCommentDialog"
       :filingId="currentFilingId"
       @close="hideCommentDialog($event)"
       attach="#todo-list"
     />
 
-    <confirm-dialog
+    <ConfirmDialog
       ref="confirm"
       attach="#todo-list"
     />
 
-    <confirm-dialog
-      ref="confirmCancelPaymentDialog"
```

Comment:
We don't need a duplicate ConfirmDialog with a different ref; we can just use the same one.